app/views/setup/_polling.html.slim

```diff
@@ -0,0 +1,4 @@
+= content_for :page_javascript do
+  javascript:
```

Comment:
turbolinks make a mess of this. Javascript keeps running in the background and after visiting another page and coming back to this, a new polling loop is started. I suggest we remove turbolinks. I could never make it work in anything other than a demo. @mssola @MaximilianMeister @ereslibre